.pre-commit-config.yaml

```diff
@@ -71,10 +71,13 @@ repos:
       - id: cspell
         name: Spell check with cspell
 
-  - repo: https://github.com/jsh9/pydoclint
-    rev: 0.5.1
+  - repo: https://github.com/pycqa/flake8
```

Comment:
This is the big problematic:
- no pinning and automatic update of pydoclint itself
- flake8 maintenance "style".
- name of hook make is very hard to call it directly (a lot to type)

Reply:
I recognise the compromise here and am open to suggestions.

I do think it is important that what ever we do there is a corresponding extension for VsCode and feedback during development.  

I wanted to bring attention to this as it is somewhat related, there is an ongoing conversation withing the ruff repo related to pydoclint. Hopefully the configuation here is short lived and ruff gets parity. https://github.com/astral-sh/ruff/issues/458#issuecomment-2122773827